packages/office-ui-fabric-react/src/components/Pivot/Pivot.tsx

```diff
@@ -108,7 +108,7 @@ export class Pivot extends React.Component<IPivotProps, IPivotState> {
     }
 
     return (
-      <a
+      <button
```

Comment:
Small request: Change any mention of "link" to "button". For example, make the function `_renderButton()` instead of `_renderLink()`.